docs/spark-standalone.md

```diff
@@ -243,6 +243,33 @@ SPARK_MASTER_OPTS supports the following system properties:
     receives no heartbeats.
   </td>
 </tr>
+<tr>
+  <td><code>spark.worker.resource.{resourceName}.amount</code></td>
+  <td>(none)</td>
+  <td>
+    Amount of a particular resource to use on the worker.
+  </td>
+</tr>
+<tr>
+  <td><code>spark.worker.resource.{resourceName}.discoveryScript</code></td>
+  <td>(none)</td>
+  <td>
+    Path to resource discovery script, which is used to find a particular resource while worker starting up.
+    And the output of the script should be formatted like <code>{"name": "gpu", "addresses": ["0","1"]}</code>.
+  </td>
+</tr>
+<tr>
+  <td><code>spark.worker.resourcesFile</code></td>
+  <td>(none)</td>
+  <td>
+    Path to resources file which is used to find various resources while worker starting up.
+    The content of resources file should be formatted like <code>[[{"id":{"componentName":
+    "spark.worker","resourceName":"gpu"},"addresses":["0","1","2"]}]]</code>.
```

Comment:
similar here do we want to say formatted like the ResourceAllocation to be consistent.

Reply:
updated, but I realize that `ResourceAllocation` is private. Maybe, needs to make it public for user ?

Reply:
its really still only used internal to Spark right now but the format of the Json has to match that.  Unless someone is going to use ResourceAllocation to generate the json file I think we can leave it private for now. We can always open it up later if needed.

Reply:
Actually thinking about this more, I guess if our comment references ResourceAllocation  they won't be able to find it in the scala docs which wouldn't be very convenient.   And I guess since the end user would be using it as an api for the resourcesFile  we should make it public then.  I would put an @evolving tag on it.

Reply:
sounds fine.